### PR TITLE
DDF-3131 Update Search-Ui-App to allow overrides

### PR DESCRIPTION
**Pull request by AzGoalie**
#### What does this PR do?
Allows search ui configuration to be overridden in the /etc directory
#### Who is reviewing it? 
@brendan-hofmann @vinamartin @emmberk 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@andrewkfiedler 
@coyotesqrl
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by AzGoalie** [comment]
build now

File: catalog/ui/search-ui-app/src/main/resources/features.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <feature name="catalog-ui" install="auto" version="${project.version}" description="Catalog UI">
         <feature prerequisite="true">camel-http</feature>
         <feature prerequisite="true">search-ui-app</feature>
-        <configfile finalname="/etc/org.codice.ddf.catalog.ui.config.config" override="true">
+        <configfile finalname="/etc/org.codice.ddf.catalog.ui.config.config" override="false">
```
Comment (emmberk) [inline catalog/ui/search-ui-app/src/main/resources/features.xml]: The commit message is a little confusing. Something like "Update catalog-ui feature to not override config file in /etc" would be clearer.

Comment (AzGoalie) [inline catalog/ui/search-ui-app/src/main/resources/features.xml]: Updated

**Comment by AzGoalie** [comment]
build now

**Comment by AzGoalie** [comment]
build now

**Pull request merged**